flixel/addons/effects/FlxRainbowSprite.hx

```diff
@@ -1,90 +0,0 @@
-package flixel.addons.effects;
-import flixel.FlxSprite;
-import flixel.graphics.FlxGraphic;
-import flixel.math.FlxMath;
-import flixel.util.FlxColor;
-import openfl.display.BitmapData;
-
-/**
- * This is a FlxSprite that copies a target FlxSprite and creates a color-cycling effect, using the target's graphic as a mask.
- * Usage: Create a new FlxSprite, as normal, and position it where you want. Create a new FlxRainbowSprite and pass it your target,
- * and add it to the state. 
- * @author Tim Hely / tims-world.com
- */
-class FlxRainbowSprite extends FlxSprite
```

Comment:
https://github.com/HaxeFlixel/flixel-addons/pull/205#issuecomment-151832324
I removed them, but it caused conflicts so I commited it back